cds/modules/records/utils.py

```diff
@@ -54,7 +53,7 @@ def is_deposit(record):
 
 def get_user_provides():
     """Extract the user's provides from g."""
-    return [need.value for need in g.identity.provides]
+    return [need.value.lower() for need in g.identity.provides]
```

Comment:
The tests are failing because some values of this `g.identity.provides` can be integers, so `lower()` will fail on those.